Make.defs

```diff
@@ -2,18 +2,29 @@ CC = gcc
 INSTALL = install -p
 MKDIR = mkdir
 
+# Flag to enable -Werror on build.
+WERROR =
+
 DBGFLAG = #-ggdb
 ifndef CFLAGS
 CFLAGS = -O2
 endif
-CFLAGS += $(DBGFLAG) -I../include -Wall
+CFLAGS += $(DBGFLAG) -I../include
+CFLAGS += -W -Wall -Wextra -Wpedantic
+ifneq ($(WERROR),)
+CFLAGS += -Werror
+endif
 #CFLAGS += -DFAIL_RECORD
 LDFLAGS += $(DBGFLAG) -L../libs
 
+PREFIX = /usr
 SBINDIR = /sbin
 SLIBDIR = /lib
-LIBDIR = /usr/lib
-LIBEXECDIR = /usr/libexec
-MANDIR = /usr/man
+INCLUDEDIR = $(PREFIX)/include
+LIBDIR = $(PREFIX)/lib
+LIBEXECDIR = $(PREFIX)/libexec
+MANDIR = $(PREFIX)/man
```

Comment:
I bet all users of tcb redefine `MANDIR` because /usr/man is not the place where manual pages are stored nowadays.  I suggest updating the value, maybe in a separate commit.

Reply:
Will fix on rebase.

Reply:
Fixed.